HACKING.md

```diff
@@ -51,6 +51,8 @@ environment variables:
 `make all` command can be used to triger all the tasks above - build the 
 Docker images, tag them and push them to the configured repository.
 
+During the make, Maven is used for packaging Java based applications (i.e. Cluster Operator, Topic Operator, ...) and the `mvn` command can be customized setting the `MVN_ARGS` environment variable before launching the `make all`. For example, it can be used to avoid running the unit tests just setting `MVN_ARGS=-DskipTests`.
```

Comment:
I know this isn't user doc, but using 'i.e.' is not compatible with the docs style guide. So we should all get into the habit of using 'that is, ' instead.

```
`make` invokes Maven for packaging Java based applications (that is, Cluster Operator, Topic Operator, ...). 
The `mvn` command can be customized by setting the `MVN_ARGS` environment variable when launching `make all`. 
For example, `MVN_ARGS=-DskipTests make all` can be used to avoid running the unit tests.
```